src/scripts.js

```diff
@@ -46,69 +59,102 @@ function displayAllRecipes() {
 }
 
 function viewRecipeDetail(event) {
-    //GOAL: incoporate instructions and total cost into recipe details page
-    const foundRecipe = recipeRepository.recipes.find((current) => {
-        return current.id === findId(event);
+    show(saveRecipeButton)
+    hide(allRecipes);
+    hide(filterSidebar);
+    show(singleRecipe);
+    show(ingredientSidebar);
+    foundRecipe = recipeRepository.recipes.find((current) => {
+        return current.id === findId(event)
```

Comment:
We have both used lines 67 and 68 in multiple functions. Should we make this a helper function and then invoke it in other functions? I thought we created it and moved it to helper functions section, but not sure we have actually been invoking it from there. Let's check that today.